### DISCUSSION TITLE
when and where to close inputstream

### DISCUSSION THREAD

**Discussion by junglechange** [line/armeria#5520] [Q&A] [answered]
GrpcMessageMarshaller.java:151 
ByteStreams.copy(method.streamResponse(message), os);
when and where to close this inputstream produced by `method.streamResponse(message)`

**Comment by ikhoon** [comment]
The standard gRPC implementation uses `ByteArrayInputStream` or `ProtoInputStream` that do not need to close. That said, it would be better to close the `InputStream`.

**Comment by ikhoon** [comment, marked as answer]
https://github.com/line/armeria/pull/5523 PR will fix the problem. Thanks for reporting. 🙇‍♂️
